src/app/Main/Body/QueryPage/Results/Result-Content.component.tsx

```diff
@@ -0,0 +1,56 @@
+/** @jsx jsx */
+import { css, jsx } from "@emotion/core";
+import { useContext } from "react";
+import { ResultsTableComponent as ResultsTable } from "./Results-Table.component";
+import { ExplanationTextComponent as ExplanationText } from "../../../../Shared/ExplanationText.component";
+import { TableContext } from "../../../../Shared/Context/TableContext";
+
+const dataTableStyle = css`
+    overflow: auto;
+`;
+
+interface ContentInterface {
+    displayRowCol: {
+        isCol: boolean;
+        isRow: boolean;
+    };
+    columnAttributes: string[];
+    getSize: (

```

Comment:
reject: this does not get the size, it sets the state of a parent component. It bypasses the normal dataflow so at least the name should reflect that.
Consider removing this callback and try to solve the problem in more common way

Reply:
I already tried to solve it in another way. But I didn't get the result that the font is centered over the corresponding rows/columns (and not the whole space) AND only the inner table is scrolled in case of an overflow.

Reply:
if you like we can discuss this issue next year. Or i can try and fix this for you. But it would be could to create a refactoring ticket for this.

Reply:
Yes discussing it next year would be great! I make a ticket.

Reply:
thx